internal/string.h

```diff
@@ -82,6 +82,8 @@ VALUE rb_setup_fake_str(struct RString *fake_str, const char *name, long len, rb
 RUBY_SYMBOL_EXPORT_END
 
 VALUE rb_fstring_new(const char *ptr, long len);
+void rb_gc_free_fstring(VALUE obj);
+void Init_fstring_table();
```

Comment:
Use the prototype declaration.

```suggestion
void Init_fstring_table(void);
```